blocks/pocketguide/pocketguide.css

```diff
@@ -0,0 +1,44 @@
+.pocketguide {
+    text-align:center;
+    position:relative;
+    padding-top:136px;
+    border-radius:10px;
+    border:1px solid #7f5732;
+    margin:100px 15px;
+    min-width: 200px;
+}
+
+.pocketguide>*>*>h2 {
+    text-transform: none;
```

Comment:
this should not be needed anymore, styles.css does not set text transform for h2

Reply:
true, sorry didnt see this copy&waste part

Reply:
it does, see styles.css:285

Reply:
I think @cziegeler removed that one - hence, we can remove here as well for now.